protocol/x/clob/types/match_perpetual_liquidation.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import (
+	gometrics "github.com/armon/go-metrics"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
+)
+
+func (m MatchPerpetualLiquidation) GetMetricLabels(callback string) []gometrics.Label {
+	return []gometrics.Label{
+		metrics.GetLabelForStringValue(
+			metrics.Callback,
```

Comment:
Why add this as a label? Seems like it should go in metric name since it's always `DeliverTx`

Reply:
removing this